toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/panels/RecorderPanel.java

```diff
@@ -0,0 +1,74 @@
+package io.toast.tk.agent.ui.panels;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.toast.tk.agent.config.AgentConfigProvider'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)